torch/_export/passes/collect_tracepoints_pass.py

```diff
@@ -0,0 +1,50 @@
+import operator
+
+import torch
+
+from torch.ao.quantization.fx._decomposed import quantized_decomposed_lib  # noqa: F401
+from torch.fx.passes.infra.pass_base import PassBase, PassResult
+
+__all__ = ["CollectTracepointsPass"]
+
+
+class CollectTracepointsPass(PassBase):
+    """
+    Performs constant folding and constant propagation.
+    """
+
+    def __init__(self, specs) -> None:
+        super().__init__()
+        self.specs = specs
+
+    def call(self, gm):
+        for module in gm.modules():
+            if not isinstance(module, torch.fx.GraphModule):
+                continue
+            for node in module.graph.nodes:
+                if node.op != "call_function":
+                    continue
+                if node.target == torch.ops.higher_order.export_tracepoint:
```

Comment:
This kind of implies that export_tracepoint is public?

Reply:
ok I will add a _